app/serializers/submission_serializer/sections/case_details.rb

```diff
@@ -1,28 +1,31 @@
 module SubmissionSerializer
   module Sections
     class CaseDetails < Sections::BaseSection
-      # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
+      # rubocop:disable Metrics/MethodLength, Metrics/AbcSize, Metrics/BlockLength
       def to_builder
         Jbuilder.new do |json|
           next unless kase
 
           json.case_details do
+            unless crime_application.non_means_tested?
+              json.case_type case_type
+              json.appeal_original_app_submitted kase.appeal_original_app_submitted
+              json.appeal_financial_circumstances_changed kase.appeal_financial_circumstances_changed
+              json.appeal_with_changes_details kase.appeal_with_changes_details
+              json.appeal_lodged_date kase.appeal_lodged_date
+              json.appeal_maat_id kase.appeal_maat_id
+              json.appeal_usn kase.appeal_usn
+              json.appeal_reference_number kase.appeal_reference_number
+              json.is_client_remanded kase.is_client_remanded
+              json.date_client_remanded kase.date_client_remanded
+            end
```

Comment:
non means are not asked case type (or any subsequent case type related questions)
non means are not asked if client remanded